prestodb/cdh5-hive/Dockerfile

```diff
@@ -59,4 +59,4 @@ EXPOSE 9083 10000
 # SOCKS PORT
 EXPOSE 1180
 
-CMD supervisord -c /etc/supervisord.conf
+CMD echo "127.0.0.1 hadoop-master" >> /etc/hosts && supervisord -c /etc/supervisord.conf
```

Comment:
Please undo this change.

Docker images are run from docker-compose, please see `presto-product-tests/conf/docker/singlenode/compose.sh config` as an example

To change the docker image for haddop there you need to:
```
 export HADOOP_MASTER_IMAGE='teradatalabs/cdh5-hive:23'
```

Reply:
discussed this offline, OK
this was needed in order to run this image with plain `docker run` command, dropping the change